workshops/vue/minis/mini6.md

```diff
@@ -432,8 +439,7 @@ The JSON structure we'll use looks like this, but we're going to load it from an
       "answers": [2, 3, 4, 5],
       "img": "https://media1.giphy.com/media/26BRzozg4TCBXv6QU/giphy.gif"
     },
-    ...
-    {
+    ...{
```

Comment:
revert this change, the linter has picked this ellipsis up as a spread operator, but that's incorrect, the ellipsis should be on it's own line.